geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -47,18 +47,20 @@
                 <div class="img" id="{{ image_info.label }}" data-controller="iiif" data-iiif-target="imageContainer">
                     <div class="img-header" data-iiif-target="imageHeader">
                         <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
-                        <input data-iiif-target="zoomSlider" data-action="iiif#handleDeepZoom" id="zoom-slider-{{ forloop.counter0 }}" type="range" name="zoom-slider" min="1" max="100" value="0" step="0.01" />
-                        <label data-iiif-target="zoomSliderLabel" for="zoom-slider-{{ forloop.counter0 }}">100%</label>
-                        <input data-iiif-target="zoomToggle" data-action="iiif#handleDeepZoom" id="zoom-toggle-{{ forloop.counter0 }}" type="checkbox" name="zoom-toggle" />
-                        <label for="zoom-toggle-{{ forloop.counter0 }}">{% translate 'Zoom and Rotate' %}</label>
-                        <button type="button" data-iiif-target="rotateLeft" class="rotate" aria-label="{% translate 'Rotate left' %}" disabled>
-                            <i class="ph-arrow-counter-clockwise"></i>
-                        </button>
-                        <button type="button" data-iiif-target="rotateRight" class="rotate" aria-label="{% translate 'Rotate right' %}" disabled>
-                            <i class="ph-arrow-clockwise"></i>
-                        </button>
+                        {% if not image_info.not_in_document %}
```

Comment:
a one-line comment here would be helpful to indicate why this section is conditional; something like this

```suggestion
                        {# only images that are part of the current document are zoomable #}
                        {% if not image_info.not_in_document %}
```